tests/testPupilCoordinates.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import division
+from builtins import zip
+from builtins import range
+from past.utils import old_div
```

Comment:
When I just look at the 'files changed' tabs above, I see no occurrences of old_div